src/panels/config/lovelace/resources/dialog-lovelace-resource-detail.ts

```diff
@@ -0,0 +1,248 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import "../../../../components/ha-dialog";
+import { HomeAssistant } from "../../../../types";
+import {
+  LovelaceResource,
+  LovelaceResourcesMutableParams,
+} from "../../../../data/lovelace";
+import { LovelaceResourceDetailsDialogParams } from "./show-dialog-lovelace-resource-detail";
+import { PolymerChangedEvent } from "../../../../polymer-types";
+
+@customElement("dialog-lovelace-resource-detail")
+export class DialogLovelaceResourceDetail extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _params?: LovelaceResourceDetailsDialogParams;
+  @property() private _url!: LovelaceResource["url"];
+  @property() private _type!: LovelaceResource["type"];
+  @property() private _error?: string;
+  @property() private _submitting = false;
+
+  public async showDialog(
+    params: LovelaceResourceDetailsDialogParams
+  ): Promise<void> {
+    this._params = params;
+    this._error = undefined;
+    if (this._params.resource) {
+      this._url = this._params.resource.url || "";
+      this._type = this._params.resource.type || "module";
+    } else {
+      this._url = "";
+      this._type = "module";
+    }
+    await this.updateComplete;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._params) {
+      return html``;
+    }
+    const urlInvalid = this._url.trim() === "";
+    // tslint:disable-next-line: prettier
+    const title = html`
+      ${this._params.resource
+        ? this._params.resource.url
+        : this.hass!.localize(
+            "ui.panel.config.lovelace.resources.detail.new_resource"
+          )}
+      <paper-icon-button
+        aria-label=${this.hass.localize(
+          "ui.panel.config.lovelace.resources.detail.dismiss"
+        )}
+        icon="hass:close"
+        dialogAction="close"
+        style="position: absolute; right: 16px; top: 12px;"
+      ></paper-icon-button>
+    `;
+    return html`
+      <ha-dialog
+        open
+        @closing="${this._close}"
+        scrimClickAction=""
```

Comment:
There is no reason to set it to an empty string. You can just remove that part
```suggestion
        scrimClickAction
```